src/Chirp.Razor/MathUtil.cs

```diff
@@ -0,0 +1,11 @@
+using Chirp.Core.Classes;
+
+namespace Chirp.Razor;
+
+public class MathUtil
```

Comment:
does this have to be its own class?

Reply:
No good point 🤣. See db3a9a0e6027294c887fa01a4adccfba676786ba